pycds/functions.py

```diff
@@ -29,6 +29,7 @@
 """
 
 from sqlalchemy.schema import DDL
+from pycds import get_schema_name
```

Comment:
There is a set of bugs, or at least shortcomings, introduced in these function definitions. The assumption we are now (mostly) making is that the search path is not to be relied on, and schema name is explicit everywhere. Except that is not the case here. These definitions were copied direct from the CRMP database definitions and they didn't include explicit schema names.

A couple of examples noted in comments below.